cmd/install/assets/hypershift_operator.go

```diff
@@ -1495,27 +1498,36 @@ func (o HyperShiftReaderClusterRole) Build() *rbacv1.ClusterRole {
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name: "hypershift-readers",
+			Labels: map[string]string{
+				"managed.openshift.io/aggregate-to-dedicated-readers": "true",
```

Comment:
Having the names of roles that are used in one deployment of HyperShift hard-coded here seems like the wrong choice - perhaps expose which roles this should aggregate into as an option for the install command.

Reply:
The role is only meant to be used for the managed use case, so I don't think we need to complicate the UX by adding more parameters. Looks fine as is.